Templates/EventHubTrigger-Python/metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "defaultFunctionName": "EventHubTriggerCSharp",
+    "description": "A Python function that will be run whenever an event hub receives a new event",
+    "name": "EventHubTrigger - Python",
+    "language": "Python",
+    "category": [
```

Comment:
Should remove core & data processing and instead use "experimental" since Python is in experimental still.